tests/integration/client/client_test.go

```diff
@@ -1,130 +0,0 @@
-// Copyright 2016 The etcd Authors
```

Comment:
Can we delete it without explicit decision about removing the discovery protocol, 
as the client probably is being used to register during the discovery process. 

Depending on v3.5 client module from v3.6 server code might create conflicts on overlapping parts of code
(but is also path to explore).

Reply:
Not sure what you mean, should client integration tests remain or not? For discovery protocol I think that we will have e2e tests.